<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{

background: qlineargradient(x1:0, y1:1, x2:0, y2:0, 
    stop:0 #add8e6, stop:1 #ffffff);
}</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;CONFIRMATION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirmationLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Enter your password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmationRemoveBTN">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#confirmationRemoveBTN{

 background-color: #FAA0A0;
font: 10pt &quot;Arial&quot;;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmationCancelBTN">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

font: 10pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
